### PR TITLE
Fix Typo in Alias Message

### PR DESCRIPTION
**Pull request by sarthakaggarwal97**
*Issue:* #1123 

*Description of changes:* Fixes a small type in the alias message

*CheckList:*
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/index-management/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: src/main/kotlin/org/opensearch/indexmanagement/indexstatemanagement/step/rollover/AttemptRolloverStep.kt [hunk 1]
```diff
@@ -415,10 +415,10 @@ class AttemptRolloverStep(private val action: RolloverAction) : Step(name) {
             "This index has already been rolled over using this alias, treating as a success [index=$index, alias=$alias]"
 
         fun getSuccessCopyAliasMessage(index: String, newIndex: String) =
-            "Successfully rolled over and copied alias from [index=$index] to [index=$newIndex]"
+            "Successfully rolled over and copy alias from [index=$index] to [index=$newIndex]"
```
Comment (bowenlan-amzn) [inline src/main/kotlin/org/opensearch/indexmanagement/indexstatemanagement/step/rollover/AttemptRolloverStep.kt:418]: This probably can still be copied?

Comment (sarthakaggarwal97) [reply src/main/kotlin/org/opensearch/indexmanagement/indexstatemanagement/step/rollover/AttemptRolloverStep.kt]: yeah, can be

**Review by r1walz — APPROVED** [review]
LGTM. ty.

**Comment by r1walz** [comment]
Hi, @sarthakaggarwal97. Thanks for contributing. Would love to see it get backported to 2.x as well. Can you take it up?

**Pull request merged**